src/components/NavBar/SearchBar.tsx

```diff
@@ -210,14 +234,14 @@ export const SearchBarDropdown = ({ toggleOpen, tokens, collections, hasInput }:
               headerIcon={<TrendingArrow />}
             />
           )}
-          {(trendingCollections?.length ?? 0) > 0 && !isTokenPage && (
+          {(trendingCollections?.length ?? 0) > 0 && !isTokenPage && phase1Flag === Phase1Variant.Enabled && (
```

Comment:
can you collapse this to a descriptive variable and `Boolean()` cast it for `render` usage?

Reply:
I'm apprehensive about creating a boolean only used here that obfuscates a lot of the context especially when 1/3 of the conditions will be removed when we remove the feature flag

Reply:
The risk here is that you can end up accidentally rendering truthy values into your template when you didn't intend to. 

i.e., 
```
const val1 = 1
const val2 = false
return val1 && val2 && <div>some text</div>
```
will render the digit ` to your template because of the sequential boolean execution

if you do this instead, you get a more descriptive template and no risk of ever accidentally rendering truthy values:
```
const val1 = 1
const val2 = false
const theTextShouldRender = Boolean(val1 && val2) // false
return theTextShouldRender  && <div>some text</div>
```

Reply:
In this case all condition are explicitly bools to avoid the above issue, notably the # of collections is explicitly compared to 0 as opposed to assuming `0` will be treated as false.

Is there some standard practice for when to collapse conditions to a `shouldShowX`? 3 or more conditions? Even if 1 of those 3 conditions is temporary when we remove the feature flag?

Reply:
I've gone with:
```
const trendingTokensLength = phase1Flag === NftVariant.Enabled ? (isTokenPage ? 3 : 2) : 4

  const trendingTokens = useMemo(() => {
    return trendingTokenResults?.slice(0, trendingTokensLength)
  }, [trendingTokenResults, trendingTokensLength])